extensions/objects/templates/FilepathEditor.js

```diff
@@ -161,6 +161,12 @@ oppia.directive('filepathEditor', [
           $scope.filepaths = response.data.filepaths;
           $scope.filepathsLoaded = true;
         });
+        $scope.width = null;
```

Comment:
Oh sorry, I meant that the app would still show the correct image size in the RTE (but it's not adjustable by the user) -- so, the changes to the width/height only get shown in the preview etc. but not in the modal.

I don't think we should have have a separate input box for height; it can just scale according to width. But for (a) the option would be to just enter a number, yep. It's not terribly intuitive, though, which is why I lean towards (b) as well.